src/resources.scss

```diff
@@ -1,3 +1,5 @@
+@import "~breakpoint-sass/stylesheets/breakpoint";
```

Comment:
Just for clarification (again): `resources.css` is intended to be used by consuming apps and `styles.css` is for internal use?

Reply:
No, they're both meant to be used by consuming apps but `resources.scss` is optional. It contains scss mixins that are not necessary for the DS to work.